testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -556,6 +556,20 @@ public void testMaterializedView()
                         "FROM\n" +
                         "  nation");
 
+        // information_schema.tables (no filtering on table_name so that ConnectorMetadata.listViews is exercised)
+        assertThat(query("SELECT table_name, table_type FROM information_schema.tables WHERE table_schema = '" + schemaName + "'"))
+                .containsAll("VALUES (VARCHAR '" + viewName + "', VARCHAR 'BASE TABLE')");
+
+        // information_schema.views
+        assertThat(computeActual("SELECT table_name FROM information_schema.views WHERE table_schema = '" + schemaName + "'").getOnlyColumnAsSet())
+                .doesNotContain(viewName);
+        assertThat(query("SELECT table_name FROM information_schema.views WHERE table_schema = '" + schemaName + "' AND table_name = '" + viewName + "'"))
+                .returnsEmptyResult();
+
+        // materialized view-specific listings
+        assertThat(query("SELECT name FROM system.metadata.materialized_views WHERE catalog_name = '" + catalogName + "' AND schema_name = '" + schemaName + "'"))
```

Comment:
are there any other columns that we should test?

Reply:
i chose `name` only because i focus on the listing ("is or isn't" there). we can revisit as needed